kura/org.eclipse.kura.container.provider/OSGI-INF/metatype/org.eclipse.kura.container.provider.ContainerInstance.xml

```diff
@@ -161,8 +161,8 @@
          </AD>
             
        <AD id="container.loggerParameters" name="Logger Parameters"
-            description="Used to pass logger parameters to a container's logging driver. Example: max-buffer-size=5m, max-file=2." type="String" cardinality="1"
-            required="false" default="max-buffer-size=5m, max-file=2" />
+            description="Used to pass logger parameters to a container's logging driver. Example: max-size=5m, max-file=2." type="String" cardinality="1"
+            required="false" default="max-size=10m" />
```

Comment:
Shouldn't we keep the default consistent with the example provided in the description?

```suggestion
            description="Used to pass logger parameters to a container's logging driver. Example: max-size=10m, max-file=2." type="String" cardinality="1"
            required="false" default="max-size=10m" />
```